docs/recipes/running-tasks-in-series.md

```diff
@@ -1,66 +1,59 @@
 # Running tasks in series, i.e. Task Dependency
 
-By default, tasks run with maximum concurrency -- e.g. it launches all the tasks at once and waits for nothing. If you want to create a series where tasks run in a particular order, you need to do two things:
-
-- give it a hint to tell it when the task is done,
-- and give it a hint that a task depends on completion of another.
-
-For these examples, let's presume you have two tasks, "one" and "two" that you specifically want to run in this order:
-
-1. In task "one" you add a hint to tell it when the task is done. Either take in a callback and call it when you're done or return a promise or stream that the engine should wait to resolve or end respectively.
-
-2. In task "two" you add a hint telling the engine that it depends on completion of the first task.
-
-So this example would look like:
+By default, tasks run with maximum concurrency -- e.g. it launches all the tasks at once 
+and waits for nothing. If you want to create a series where tasks run in a particular order, 
+you should use `gulp.series`;
 
 ```js
 var gulp = require('gulp');
 
 // takes in a callback so the engine knows when it'll be done
-gulp.task('one', function(cb) {
-    // do stuff -- async or otherwise
-    cb(err); // if err is not null and not undefined, the orchestration will stop, and 'two' will not run
+gulp.task('one', function(done) {
+  // do stuff -- async or otherwise
+  // if err is not null and not undefined, the task fails 
+  // and 'two' will not run
+  done(err); 
 });
 
 // identifies a dependent task must be complete before this one begins
-gulp.task('two', ['one'], function() {
-    // task 'one' is done now
+gulp.task('two', gulp.series('one', function(done) {
+  done();
 });
 
-gulp.task('default', ['one', 'two']);
-// alternatively: gulp.task('default', ['two']);
+gulp.task('default', gulp.parallel('two'));
 ```
 
-Another example, which returns the stream instead of using a callback:
-
+Another example, which returns the stream and promise instead of using a callback.
+It also uses `once` to run dependencies once:
 ```js
 var gulp = require('gulp');
-var del = require('del'); // rm -rf
+var promisedDel = require('promised-del'); // rm -rf
+var once = require('once');
 
-gulp.task('clean', function(cb) {
-    del(['output'], cb);
-});
+gulp.task('clean', once(function() {
+  // The clean task will be run twice, but this function will be run once.
+  // So we cannnot rely on a callback.
```

Comment:
I think we could do away with the series recipe contents and just add a link that points to the docs for `series`.

I agree about doing away with the "dependency" terminology because everything is about function composition now.

I think we should have a "running once" recipe because people have come to expect that behavior from current gulp.  The recipe just makes the upgrade path nicer.